.github/workflows/build_deb_packages.yml

```
@@ -27,7 +27,7 @@ jobs:
 
       - name: Fetch debops-ci
         run: |
-          wget https://raw.githubusercontent.com/ooni/sysadmin/master/tools/debops-ci
+          wget https://raw.githubusercontent.com/ooni/sysadmin/fix-debops/tools/debops-ci
```

I don't think we should merge this code. I approved the corresponding PR in ooni/sysadmin. You should probably merge such a pull request and then apply the following suggestion:

```suggestion
          wget https://raw.githubusercontent.com/ooni/sysadmin/master/tools/debops-ci
```